.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
   "mochaExplorer.require": [
     "ts-node/register"
   ],
+  "mochaExplorer.debuggerConfig": "Debug Mocha Tests",
   "solidity.compileUsingRemoteVersion": "v0.8.11+commit.d7f03943",
 }
```

Comment:
nit: will be nice if you configure Vscode to add new line at end of file automatically, so those red guys disaapear
like so https://stackoverflow.com/a/54270768/3139595